capacity4more/behat/features/group_status.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Group Status
+  Test group status field
+
+  @api
+  Scenario: Check requested group as admin
+    Given I am logged in as user "mariecurie"
+    When  I start editing group "Requested group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "archived"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+
+  @api
+  Scenario: Check requested group as group owner
+    Given I am logged in as user "alfrednobel"
+    When  I start editing group "Requested group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "rejected"
+    And   the "#edit-c4m-og-status-und" element should not contain "draft"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+    And   the "#edit-c4m-og-status-und" element should not contain "archived"
+
+  @api
+  Scenario: Check rejected group group as admin
+    Given I am logged in as user "mariecurie"
+    When  I start editing group "Rejected group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "requested"
+    And   the "#edit-c4m-og-status-und" element should not contain "draft"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+    And   the "#edit-c4m-og-status-und" element should not contain "archived"
+
+  @api
+  Scenario: Check rejected group group as group owner
+    Given I am logged in as user "alfrednobel"
```

Comment:
@HelenaEksler A group owner should get access denied when a group is requested, rejected or deleted